src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -241,6 +241,7 @@ class Microsoft::Terminal::Core::Terminal final :
     void SetBlockSelection(const bool isEnabled) noexcept;
     void UpdateSelection(SelectionDirection direction, SelectionExpansion mode);
     void SelectAll();
+    const bool IsInQuickEditMode() const noexcept;

```

Comment:
There are no `const` return values. It's entirely up to the caller to decide if it's `const` or not.